Resources/doc/reference/architecture.rst

```diff
@@ -325,7 +325,7 @@ To do this, you first need to call the ``addChild`` method in your PostAdmin ser
     .. code-block:: xml
 
         <!-- app/config/config.xml -->
-        <service id="sonata.news.admin.post" class="Sonata\NewsBundle\Admin\PostAdmin">
+        <service id="sonata.news.admin.post" class="AppBundle\Admin\PostAdmin">
```

Comment:
i would not touch this line and keep `Sonata\NewsBundle\Admin\PostAdmin`, because this is the initial step to enable the PostAdmin and add a note afterwards that you need to create/use a custom Admin in `AppBundle\Admin` if you would like to modify ne menu for example

what do you think @Soullivaneuh